### PR TITLE
Update user.md with clearer usage instructions

### PR DESCRIPTION
**Pull request by malak-coforma**

### CODE REVIEW

File: user.md [hunk 1]
```diff
@@ -1,32 +1,229 @@
...
+
+If the response returns an error, make sure you are logged in as either a Government Care Systems Coordinator or an EHR Maintainer with a matching `facilityNpi` that corresponds with the user.
+
+#### POST /patientStatus/
+
+Accepts an update for a patient. Requires the facility's NPI, the facility's patient ID, the ID of the disaster impacting the patient, the ID of their current status, and the date that this update was recorded on. On success, the endpoint will return the API's generated ID for the patient.
+
+**Only EHR role users have permission to create a new patient status. Other users are unauthorized.**
+
+##### Sample request body
```
Comment (bodnarbm) [inline user.md:103]: Correctly as implemented in the code, it's not a request body, but provided as query parameters. There's an example curl request for this in the 04-test.sh that may help with updating this.

Comment (malak-coforma) [reply user.md:103]: Ah, ok, so I should update to:

##### Sample request parameters

and then put it in a table like I did for the other GET parameters?

**Review by gmrabian — APPROVED** [review]
good pending the changes Brandon mentioned

File: user.md (hunk 1 above)
Comment (bodnarbm) [reply user.md:103]: I think that makes sense. As long as we are clear they are query parameters and that it's a POST call, we should be good.

Should we add something like an example CURL command to help with clarity?

Comment (malak-coforma) [reply user.md]: I updated it to a table. I don't know that I want to add CURL commands as these instructions are based on using the SwaggerUI and I don't have instructions written for authenticating via CURL. 🤔 I don't think I could write those, and would need someone else to.

**Pull request merged**